src/App.css

```diff
@@ -550,7 +550,7 @@ a.text-link {
 
 .thought,
 .thought-annotation {
-  max-width: calc(100% - 1em); /* expand click area. offset content padding to prevent line break */
+  width: calc(100% - 1em); /* expand click area. offset content padding to prevent line break */
```

Comment:
This change has the undesirable effect of extending the click area to the right edge of the screen, which more easily interferes with scrolling and user gestures. Instead, the width of a thought should generally be limited to the width of its content, plus a little padding.

Reply:
Yes, let me use `max-width`